_posts/2015-04-03-april-newsletter.md

```diff
@@ -0,0 +1,285 @@
+that easy mode leveraged.
+
+The [evim.vim runtime][easy-runtime] file contained additional settings. The
+file was also removed but the details on what it enabled can still be viewed
+in Vim's documentation (but not in Neovim's) using [`:h
+evim-keys`][easy-evim-keys].
+
+### Merging of vimdiff and -d
+
+Similar to `evim`/`vim -y` described above, [`vimdiff`][vimdiff-docs] is the
+same as `vim -d` which starts Vim in diff mode.
+
+@Pyrohh created a [pull request][vimdiff-pr] that merged these two different
+ways of viewing a diff into just `nvim -d` in an effort to cut down on the
+[extra symlinks][vimdiff-symlinks] that Vim uses.
+
+### State of the GUI
+
+TODO
+
+### New Tab Events
+
+Three new [autocommand][events-autocmd] events have been added by @fmoralesc in
+[this pull request][events-pr]. An [autocommand event][events-list] is a way to
+execute something in response to a particular event. Events are particularly
+useful in plugins.
+
+The new events are: [`TabNew`][events-tabnew],
+[`TabNewEntered`][events-tabnewentered], and [`TabClosed`][events-tabclosed].
+
+### Major Change to Improve Job API
+
+@tarruda is continuing to refine the new job API. His work has continued in
+[this latest pull request][job-pr]. This might be a breaking change for existing
+plugins that use the existing job API.
+
+This improvement is to make use of function callbacks (for input, output events)
+instead of the previous way of using [autocommands][job-autocmd] for responding
+to events.
+
+In addition to that, there are more job control functions in VimL including
+[`jobwait()`][job-wait] and [`jobclose()`][job-close].
+
+### So Long `long_u`
+
+In an epic crusade against Vim specific integer types that has spanned two
+previous newsletters (June and November), @elmart has finished removing the
+integer type `long_u`. @elmart's changes were broken up into three last pull
+requests ([1][long-1], [2][long-2], [3][long-3]).
+
+This is a step towards
+@tarruda's goal to [remove all project specific types][long-types]. All that
+remains is `char_u` in which there are ~12,300 occurrences currently in use.
+
+### Adding a EndOfBuffer Highlight Group
+
+@mhinz wanted a way to [remove ~ after the file ends][highlight-idea]. He
+proposed a new [:highlight][highlight-docs] group, called
+[`EndOfBuffer`][highlight-endofbuffer], to go with the current [list of
+groups][highlight-groups].
+
+His [proposal][highlight-proposal] was first sent as a patch to
+[vim-dev][highlight-vim-dev] where his changes were accepted by Bram and should
+make it into Vim in the future. @mhinz let @fmoralesc prepare the [pull
+request][highlight-pr] to bring in the changes to Neovim. The pull request has
+since been merged into the master branch.
+
+### Upstream Vim Patches
+
+Neovim continues to merge in the upstream Vim patches that are submitted to the
+Vim mailing list.
+
+Since the last newsletter, [60 pull requests][patch-search] worth of patches
```

Comment:
Since this was written, I think a few more have been added, so perhaps `more than 60 pull requests ...`?